test/common/scenarios.js

```diff
@@ -57,6 +58,25 @@ function Multisig(options) {
 		passphrase: this.account.passphrase,
 		secondPassphrase: this.account.secondPassphrase,
 	});
+
+	this.transfer = (
+		recipientId,
+		passphrase = accountFixtures.genesis.passphrase,
+		amount = 100000000000
+	) => {
+		return lisk.transaction.transfer({
+			amount,
+			passphrase,
+			recipientId,
+		});
+	};
+
+	this.signTransaction = (signers, transaction) => {
```

Comment:
I don't see the point of introducing this helper because it obfuscates the process. But if you really want to do so, please use it uniformly across the file and ideally across the whole suite...